driver_scripts/setup_ocean_basins.py

```diff
@@ -18,6 +18,8 @@ def spcall(args): #{{{
 
 options, args = parser.parse_args()
 
+groupName = 'BasinRegionsGroup'
```

Comment:
The name sounded too generic and not specific enough.  Maybe something like `GlobalOceanBasinRegionsGroup` would be better?